app/server/utils/ussd/token_processor.py

```diff
@@ -91,8 +91,9 @@ def get_limit(user: User, token: Token) -> Optional[TransferLimit]:
         if len(limits) == 0:
             return None
         else:
+            # todo: fix in token agent branch
             # might want to do something different if there's more than one limit...
-            return limits[0]
+            return limits[-1]
```

Comment:
This is a quick fix to always get last limits (GE), which will be fixed in Token Agent branch